apps/explorer/src/utils/api.ts

```diff
@@ -20,6 +20,10 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { UseQueryOptions } from '@tanstack/react-query'
 
+// TODO: Trying to import the enum value from the explorer api doesn't work
+export const POST_QUERY_LIMIT = 80
+export const PAGINATION_PAGE_LIMIT = 100
+
```

Comment:
Check the last file of my WIP PR here. I added this to the shared library. This is the solution that Thomas and I agreed upon: https://github.com/alephium/alephium-frontend/pull/132/files

```ts
import { MaxSizeTokens } from '@alephium/web3/dist/src/api/api-explorer'

export const TOKENS_QUERY_LIMIT: MaxSizeTokens = 80
```

Reply:
Feel free to add it in the shared lib like I did in my PR

Reply:
Just for completion: Thomas exposed 2 enums so far:

```
export enum MaxSizeTokens {
  Value80 = 80
}

export enum MaxSizeAddresses {
  Value80 = 80
}
```